server/api/index.js

```diff
@@ -8,17 +8,60 @@ var express = require( 'express' );
  */
 var version = require( '../../package.json' ).version,
 	config = require( 'config' ),
+	request = require( 'request' ),
+	cookie = require( 'cookie' ),
+	cors = require( 'cors' ),
 	oauth = require( './oauth' );
 
+var corsOptions = {
+	origin: 'http://teach-cms-enabled.herokuapp.com',
```

Comment:
should I read from `process.env` instead? nothing will break really unless you have different version of that server...